pkg/master/client_ca_hook_test.go

```diff
@@ -189,18 +189,20 @@ func TestWriteClientCAs(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		client := fake.NewSimpleClientset(test.preexistingObjs...)
-		test.hook.tryToWriteClientCAs(client.Core())
+		t.Run(test.name, func(t *testing.T) {
+			client := fake.NewSimpleClientset(test.preexistingObjs...)
+			test.hook.tryToWriteClientCAs(client.Core())
 
-		actualConfigMaps, updated := getFinalConfiMaps(client)
-		if !reflect.DeepEqual(test.expectedConfigMaps, actualConfigMaps) {
-			t.Errorf("%s: %v", test.name, diff.ObjectReflectDiff(test.expectedConfigMaps, actualConfigMaps))
-			continue
-		}
-		if test.expectUpdate != updated {
-			t.Errorf("%s: expected %v, got %v", test.name, test.expectUpdate, updated)
-			continue
-		}
+			actualConfigMaps, updated := getFinalConfiMaps(client)
+			if !reflect.DeepEqual(test.expectedConfigMaps, actualConfigMaps) {
+				t.Errorf("%s: %v", test.name, diff.ObjectReflectDiff(test.expectedConfigMaps, actualConfigMaps))
+				return
+			}
+			if test.expectUpdate != updated {
+				t.Errorf("%s: expected %v, got %v", test.name, test.expectUpdate, updated)
+				return
```

Comment:
here too. (this is, per the article you linked, one of the reasons for doing it this way)

Reply:
Done,Thanks!